source/types/trinary.cpp

```diff
@@ -226,6 +226,17 @@ intToTrits(const int64_t& value, std::size_t length) {
   return res;
 }
 
+void
+incrementTrits(Trits& trits) {
+  for (unsigned int i = 0; i < trits.size(); ++i) {
+    if (++trits[i] > 1) {
+      trits[i] = -1;
+    } else {
+      break;
+    }
+  }
+}
+
```

Comment:
this is driving me crazy, I am sure I already saw that code somewhere in our code base, but after looking up, seems not, might be confused with another library I guess.

Reply:
It was indeed involved in a previous version of Signing::key but was removed in the optimization patch.